runtime/build.gradle

```diff
@@ -29,6 +33,11 @@ targetList.each { targetName ->
         linkerArgs project.file("../common/build/$targetName/hash.bc").path
     }
 
+    tasks.create("${targetName}Mimalloc", CompileCToBitcode, file('src/mimalloc'), "mimalloc", targetName).configure {
+        excludeFiles.addAll(["**/alloc-override*.c", "**/page-queue.c", "**/static.c"])
+        excludedTargets.addAll(['wasm32', 'linux_mips32', 'linux_mipsel32', 'zephyr'])
```

Comment:
Maybe explicit opt-in instead of opt-out?

Reply:
I did excluded list because it's smaller, I want to do this next as extension property of target, but I didn't want waiting for updating kotlin version.

Reply:
You can have an extension property in K/N. We shall opt-in only to platforms we tested allocator upon.

Reply:
If I placed it in K/N I can't use it both in K/N and Gradle.
I'll have to repeat same twice.

Reply:
I haven't managed to make it as extension, can't get it from gradle then.

Reply:
What I  plan https://github.com/JetBrains/kotlin/pull/2911
https://github.com/JetBrains/kotlin-native/pull/3709